storio-common-annotations-processor/src/test/java/com/pushtorefresh/storio/common/annotations/processor/StorIOAnnotationsProcessorDummy.java

```diff
@@ -0,0 +1,95 @@
+package com.pushtorefresh.storio.common.annotations.processor;
+
+import java.lang.annotation.Annotation;
+import java.util.Map;
+
+import javax.annotation.processing.RoundEnvironment;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.util.Elements;
+
+import com.pushtorefresh.storio.common.annotations.processor.generate.ResolverGenerator;
+import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOColumnMeta;
+import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOTypeMeta;
+import com.squareup.javapoet.JavaFile;
+import com.squareup.javapoet.TypeSpec;
+
+@SuppressWarnings("rawtypes")
+public final class StorIOAnnotationsProcessorDummy

```

Comment:
Hi @artem-zinnatullin, because StorIOAnnotationsProcessor class is abstract we need dummy class to test base class. Also in the source code we are using unimplemented methods like getTypeAnnotationClass().